tests/cephfs/cephfs_bugs/test_mds_laggy_thrown_out.py

```diff
@@ -0,0 +1,146 @@
+import secrets
+import string
+import traceback
+
+from ceph.ceph import CommandFailed
+from tests.cephfs.cephfs_utilsV1 import FsUtils
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    try:
+        """
+         CEPH-83575624 - when a standby-replay node is laggy it should be thrown out from the MDS map
+                        if the beacon grace period reaches 15sec.
+
+        Steps:
+        1. Create filesystem
+        2. mount the Fs and set mas mds 2
+        3. set allow_standby_replay to true/1
+        4. write data
+        5. collect standby reply nodes
+        6. Stop the network for 10 sec on the first stnadby reply node
+        7. verify if there is change in stnadby nodes
+        8. Stop the network for 60 sec
+        9. Verify if there change in standby reply nodes. Expectations is to have change in standby node
+
+        As part of TC delete the contents on lost+found  directory
+
+        Cleanup:
+        1. unmount the FS
+        2. Remove FS
+
+        """
+        tc = "CEPH-83575781"
+        log.info("Running cephfs %s test case" % (tc))
+        fs_util = FsUtils(ceph_cluster)
+        config = kw.get("config")
+        clients = ceph_cluster.get_ceph_objects("client")
+        build = config.get("build", config.get("rhbuild"))
+        fs_util.prepare_clients(clients, build)
+        fs_util.auth_list(clients)
+        client1 = clients[0]
+        fs_name = "cephfs_mds_laggy"
+        file = "network_disconnect.sh"
+        fs_util.create_fs(client1, fs_name)
+        mds_nodes = ceph_cluster.get_nodes("mds")
+        host_list = [node.hostname for node in mds_nodes]
+        hosts = " ".join(host_list)
+        client1.exec_command(
+            sudo=True,
+            cmd=f"ceph orch apply mds {fs_name} --placement='5 {hosts}'",
+            check_ec=False,
+        )
+        client1.exec_command(sudo=True, cmd=f"ceph fs set {fs_name} max_mds 2")
+        client1.exec_command(
+            sudo=True, cmd=f"ceph fs set {fs_name} allow_standby_replay 1"
+        )
+
+        fuse_mount_dir = "/mnt/fuse_" + "".join(
+            secrets.choice(string.ascii_uppercase + string.digits) for i in range(5)
+        )
+        fs_util.fuse_mount(
+            [client1],
+            fuse_mount_dir,
+            new_client_hostname="admin",
+            extra_params=f" --client_fs {fs_name}",
+        )
+        client1.exec_command(
+            sudo=True,
+            cmd=f"cd {fuse_mount_dir};"
+            f"mkdir subdir;"
+            f"dd if=/dev/urandom of=subdir/sixmegs bs=1M conv=fdatasync count=6 seek=0;",
+        )
+
+        standby_reply_mdss = fs_util.get_standby_reply_mdss(client1, fs_name=fs_name)
+        standby_reply_mdss_objs = [
+            ceph_cluster.get_node_by_hostname(i.split(".")[1])
+            for i in standby_reply_mdss
+        ]
+        for standby_node in standby_reply_mdss_objs:
+            standby_node.upload_file(
```

Comment:
As we are executing bash script just on  standby_reply_mdss_objs[0] node, should we copy this script on all nodes?